nsls2forge_utils/tests/test_all_feedstocks.py

```diff
@@ -0,0 +1,54 @@
+import os
+import shutil
+import subprocess
+
+from nsls2forge_utils.all_feedstocks import (
+    get_all_feedstocks_from_github,
+    get_all_feedstocks,
+    all_feedstocks_info
+)
+
+
+def test_all_feedstocks_from_github():
+    names = get_all_feedstocks_from_github()
+    assert names is None
+    names = get_all_feedstocks_from_github(organization='nsls-ii-forge')
+    archived_names = get_all_feedstocks_from_github(organization='nsls-ii-forge',
+                                                    archived=True)
+    size_names = len(names)
+    size_archived = len(archived_names)
+    assert size_archived >= size_names
+    assert size_names > 0
+    assert size_archived > 0
+
+
+def test_all_feedstocks():
+    names = get_all_feedstocks()
+    assert names is None
+    with open('names.txt', 'w') as f:
+        f.write('event-model\n')
+        f.write('bluesky')
+    names = get_all_feedstocks(cached=True)
+    assert 'event-model' in names
+    assert 'bluesky' in names
+    assert len(names) == 2
+    os.remove('names.txt')
+    os.makedirs('./test_feedstocks/event-model-feedstock', exist_ok=True)
+    os.makedirs('./test_feedstocks/bluesky-feedstock', exist_ok=True)
+    names = get_all_feedstocks(cached=True, feedstocks_dir='./test_feedstocks/')
+    assert 'event-model' in names
+    assert 'bluesky' in names
+    assert len(names) == 2
+    shutil.rmtree('./test_feedstocks')
+
+
+def test_all_feedstocks_info():
+    cmd = ('git clone --depth 1 https://github.com/nsls-ii-forge/event-model-feedstock.git '
+           './test_feedstocks/event-model-feedstock')
```

Comment:
Will it create the `test_feedstocks` dir automatically? It was removed in the previous test...

Reply:
Yes, I just tested it and it creates all directories in the path if they do not exist.

Reply:
Sounds good!